atomics/T1059.006/T1059.006.yaml

```diff
@@ -148,3 +148,23 @@ atomic_tests:
       name: sh
       cleanup_command: |
         rm #{python_binary_name} #{python_script_name} #{payload_file_name}
+  - name: 'Python pty module and spawn function used to spawn sh or bash'
+    description: |
+     ID T1059.006. Adversaries may abuse Python commands and scripts for execution. Python is a very popular scripting/programming language, with capabilities to perform many functions. Python can be executed interactively from the command-line (via the python.exe interpreter) or via scripts (.py) that can be written and distributed to different systems. Python code can also be compiled into binary executables.Python comes with many built-in packages to interact with the underlying system, such as file operations and device I/O. Adversaries can use these libraries to download and execute commands or other scripts as well as perform various malicious behaviors.Zero-Day Exploitation of Atlassian Confluence [CVE-2022-26134] Unauthenticated RCE vulnerability - Critical severity. As per Volexity, bash shells were launched by the Confluence web application process. It had spawned a bash process which spawned a Python process that in turn spawned a bash shell. Reference: https://www.volexity.com/blog/2022/06/02/zero-day-exploitation-of-atlassian-confluence
+    supported_platforms:
+    - linux
+    dependencies: 
+      - description: |
+      Verify if python is in the environment variable path and attempt to import requests library.
```

Comment:
looks like this needs to be indented, CI is failing due to it.

Reply:
Indentation adjusted. Thanks!